pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -57,6 +60,17 @@ def _apply_incubators(self):
             for egg in self.eggs:
                 if egg["used"] or egg["km"] == -1:
                     continue
+                
+                if self.breakable_incubator:
+                    if incubator.get('uses_remaining') is not None:
+                        if egg["km"] not in self.breakable_incubator:
+                            continue
+                    
+                if self.infinite_incubator:
+                    if incubator.get('uses_remaining') is None: 
```

Comment:
I meant line 70. Just curious if we needed to check that the infinite incubator had remaining uses (as I haven't looked at incubator.get. Otherwise, this is a great PR - that FR has been up for days now.